unittests/runtime/TaskStatus.cpp

```diff
@@ -10,8 +10,9 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "swift/Runtime/Concurrency.h"
+#include "Concurrency/TaskPrivate.h"
```

Comment:
Hmm.  This unit test might not link anymore if these aren't part of the public ABI.  I'm not sure it's possible to write this test at all; it might be okay to just get rid of it, though.

Reply:
Sounds good, removed it.